infra/src/main/kotlin/org/yapp/infra/readingrecord/repository/impl/JpaReadingRecordQuerydslRepositoryImpl.kt

```
@@ -31,7 +31,7 @@ class JpaReadingRecordQuerydslRepositoryImpl(
         val results = queryFactory
             .selectFrom(readingRecord)
             .where(whereCondition)
-            .orderBy(createOrderSpecifier(sort))
+            .orderBy(*createOrderSpecifiers(sort))
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)